containers/pom.xml

```diff
@@ -84,6 +75,20 @@
 
   <profiles>
     <profile>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
Please remove the default profile, it shouldn't exist. The 'default' should be to run the eap profile, just like like before.